cypress-tests/cypress/fixtures/create-confirm-body.json

```diff
@@ -27,7 +27,7 @@
     "card": {
       "card_number": "4242424242424242",
       "card_exp_month": "01",
-      "card_exp_year": "24",
```

Comment:
Is this being used anywhere? Seeing as the expiry date is 01-2024?

Reply:
it was being overriden by connector specific configs. so we never faced any issue until today from this